x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview_metrics.tsx

```diff
@@ -28,6 +28,7 @@ export const EngineOverviewMetrics: React.FC = () => {
           </h1>
         </EuiTitle>
       </EuiPageHeader>
+      <FlashMessages />
```

Comment:
I don't need this here... but it seems silly not to include this the normal view if I'm including it in the empty state view